src/api/app/controllers/webui/request_controller.rb

```diff
@@ -34,6 +35,13 @@ def show
       @open_reviews_for_staging_projects = @bs_request.reviews.opened.for_staging_projects
       @refresh = @action[:diff_not_cached]
 
+      # retrieve a list of all package maintainers that are assigned to at least one target package
+      @package_maintainers = target_package_maintainers
```

Comment:
I agree it is a relevant information. The very next question is: do we want to list the project maintainer in the same list of the package maintainers or do we want to differentiate them a little bit? And also: is the alert hint message still relevant or? it talks to the project maintainer for a different but overlapping reason.....

Reply:
We should show the package and project maintainers, but clearly distinguish them from one another. Package maintainers should be listed before project maintainers, since project maintainers are the last resort maintainers, if somehow package maintainers aren't active or available.

Reply:
Addressed in 593b10c029f2ed62fce7d10b4c12fef287d715f5

I decided to keep them split into two different list, as it is a common practice we do across the whole project,